pkgs/top-level/python-packages.nix

```diff
@@ -3974,8 +3974,6 @@ self: super: with self; {
     inherit (pkgs.darwin.apple_sdk.frameworks) Security;
   };
 
-  etebase-server = callPackage ../servers/etebase { };
```

Comment:
But do remember to add an entry in python-aliases.nix!

Reply:
Can you explain why? We have an agreement between both maintainers of the package, @felschr and myself, to drop the python package.

Reply:
We always add an entry there except for extra versions. Then we can also drop the release notes entry.

Reply:
Ah right, sorry i misunderstood the suggestion at first. Should be fixed now?